src/ASM/ASMs2DIB.h

```diff
@@ -31,7 +31,7 @@ class ASMs2DIB : public ASMs2D
     element is intersected by the immersed boundary, or if it is neighbor to
     an element that is intersected.
   */
-  class Intersected : public InterfaceChecker
+  class Intersected : public ASMs2D::InterfaceChecker
```

Comment:
if you see the downstream you will see that i do subclass because it's the application that knows where it might want to do the integrals (ie where things jump). mixed continuity splines, generic code won't know that jumps are in velocity gradient (and in theory pressure itself), etc etc. and there i do need access to the myPatch. there's a dedicated member in IB so i don't have to cast all over, i rather remove that if this offends you.

Reply:
I did not look at any downstream repos, but I believe it if you say it. It's fine, as long the ASMu2DIB member with same name does not cause any trouble in that class (how does it determine which member to use there since both are visible?)

Reply:
the current class member shadows the base class member, and you can address the base class member with baseclass:: prefix. but i changed this, and i like it better with the change. turns out i only had to cast in a single spot (into a local var) and that is a very small cost compared to the potential confusion.

Reply:
OK, only now I see that the ASMs2DIB member now is gone. Then I agree this is the best way.